plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergFailureRecoveryTest.java

```diff
@@ -148,18 +148,88 @@ public void testDelete()
                 .finishesSuccessfully();
     }
 
+    // Copied from BaseDeltaFailureRecoveryTest
```

Comment:
Let's make two different base methods for the different connector types, rather than copying a very complex test.

Reply:
@losipiuk Can you suggest a name for the modified test?